lambda_function.py

```diff
@@ -0,0 +1,54 @@
+import json
+import requests
+
+def lambda_handler(event, context):
+
+    # Base URL of Gupshup's OTP API, including all constant params
+    baseURL = "https://enterprise.smsgupshup.com/GatewayAPI/rest?userid=2000173120&password=Avanti@12345&method=TWO_FACTOR_AUTH&v=1.1&msg=%25code%25%20is%20your%20OTP%20from%20Avanti%20Fellows&format=text&otpCodeLength=4&otpCodeType=NUMERIC";
+
+    response_headers={"Access-Control-Allow-Origin" : "*"}
```

Comment:
use appropriate spacing - follow pep8 guidelines (https://www.python.org/dev/peps/pep-0008/)

Reply:
also, please set up pre-commit locally for the repo so that these issues are caught before the commit itself